charts/brig/values.yaml

```diff
@@ -78,6 +78,14 @@ config:
     # tlsCaSecretRef:
     #   name: <secret-name>
     #   key: <ca-attribute>
+
+  # Postgres connection settings.
+  # Don't forget to set the user's password in `secrets.pgPassword`
```

Comment:
Maybe metion that all https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS are posssible here, including pgpassfile.

Reply:
Sorry, I've overlooked those. Adding them.

Reply:
Okay, I've tried to explain this a bit more.

Now, I'm wondering if we really need `postgresqlPassword`. Or, if it cannot be defined via `passfile` (https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-PASSFILE) :thinking:

Reply:
Thankfully, @battermann remembered the details: Providing the password file via a volume mount won't work (we tried that.) The issue is the `brig` runs with a random uid and the library (`libpq-connect`) expects an access right mask of `0600` on the password file.

I've now added this insight to the docs.

Reply:
Yes, only `brig` runs with an unprivileged user, not a random one.